sdk/apis/kubermatic/v1/policy_binding.go

```diff
@@ -84,20 +82,35 @@ type PolicyBindingSpec struct {
 	// +kubebuilder:validation:Required
 	PolicyTemplateRef corev1.ObjectReference `json:"policyTemplateRef"`
 
-	// Enabled controls whether the policy defined by the template should be actively applied to the cluster.
+	// KyvernoPolicyNamespace specifies the Kyverno namespace to deploy the Kyverno Policy into.
 	//
 	// Relevant only if the referenced PolicyTemplate has spec.enforced=false.
+	// If Template.NamespacedPolicy is true and this field is omitted, no Kyverno Policy resources will be created.
 	//
 	// +optional
-	Enabled *bool `json:"enabled,omitempty"`
+	KyvernoPolicyNamespace *KyvernoPolicyNamespace `json:"kyvernoPolicyNamespace,omitempty"`
+}
+
+// KyvernoPolicyNamespace specifies the Kyverno namespace to deploy the Kyverno Policy into.
+// This is relevant only if a Kyverno Policy resource is created because Kyverno Policy is namespaced.
+// For Kyverno ClusterPolicy, this field is ignored.
+type KyvernoPolicyNamespace struct {
+	// Name is the namespace to deploy the Kyverno Policy into.
+	//
+	// +kubebuilder:validation:Pattern:=`^(|[a-z0-9]([-a-z0-9]*[a-z0-9])?(\\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*)`
+	// +kubebuilder:validation:MaxLength:=63
+	// +kubebuilder:validation:Type=string
+	Name string `json:"name"`
 
-	// NamespaceSelector specifies which namespaces the Kyverno Policy resource(s) should be created in
+	// Labels of the Kyverno namespace
 	//
-	// Relevant only if Template.NamespacedPolicy is true.
-	// If Template.NamespacedPolicy is true and this selector is omitted, no Kyverno Policy resources will be created.
+	// +optional
+	Labels map[string]string `json:"labels,omitempty"`
+
+	// Annotations of the Kyverno namespace
```

Comment:
Can we rephrase to "... to apply to this namespace"? To me the "kyverno namespace" is not the same as "the namespace policies are installed into" necessarily.